markets/perps-market/contracts/modules/PerpsAccountModule.sol

```diff
@@ -54,6 +54,11 @@ contract PerpsAccountModule is IAccountModule {
             account.id = accountId;
         }
 
+        // Check if there are pending orders
+        if (AsyncOrder.load(accountId).sizeDelta != 0) {
+            revert PendingOrdersExist();
+        }
```

Comment:
could move this into the account storage and do `account.checkPendingOrders()` or smth.  as much as possible trying to move validations into storage libraries.  non blocking; only if you think its cleaner